Gemfile

```diff
@@ -1,7 +1,5 @@
 source "https://rubygems.org"
 
-ruby File.read(".ruby-version").strip
```

Comment:
Should we keep this line in, ready for when the automatic update is done?

Reply:
I thought this was the line we were getting rid of because it puts the ruby version in the Gemfile.lock? Have I completely misunderstood?